@here/harp-map-theme/resources/berlin_tilezen_textured.json

```diff
@@ -0,0 +1,1169 @@
+{
```

Comment:
I know that it's out-of-scope of this task, but copying whole theme for this example is ... ovekill.

Maybe something like this can work:

1) Ensure all styles in original `berlin...json` are findable by for example unique "description".
2) Load original theme using ThemeLoader - as usual
3) Update specific techniques (like `park) like this:

```
updateTheme(theme, "park", {
    technique: "standard-textured",
    attr: {
        ...
    }
});
```

This also should work

```
function updateStyle(theme: Theme, description: string, newStyle: Style) {
    ThemeVisitor.visit(theme, style => {
         if( style.description === "description") {
              Object.assign(style, newStyle);
        }
    });
}
```

Reply:
I was also thinking about this, but since this is an example it would detract the users attention from the really important thing, i.e.:
_You can just use a texture in the stylesheet and that's it. No code changes needed_
If I would do what you propose it looks like you have to load the Theme and modify it to use textures, but that's not the case.

Maybe this discussion just reveals a generic problem we have right now: There are no simple examples for the themes. We only have a full blown theme. So in the future we might want to have really simple theme files that also help to understand our styling capabilities.

Reply:
Another idea that just came to my mind was to patch the berlin base theme and generate the textured one at build time (like prepack.ts). Than we would at least have both themes in sync. What do you think?

Reply:
Yes, the problem is big, but on the other hand, source of this example is soooo big, that it's no longer "you can use texture, here is how to do it", it is rather "here is big pile of unreadable JSON with 4 lines of real interest  hidden in 1000 lines of uncommented JSON".

IMHO, Update _here_ (i mean locally, in same source file) as proposed, is much more "local" and discoverable than non-commented JSON blob.

(this is bigger discussion, we can do it later)